www/caddy/src/opnsense/service/templates/OPNsense/Caddy/Caddyfile

```diff
@@ -48,6 +48,9 @@
         format json {
             time_format rfc3339
         }
+        {% if generalSettings.LogLevel %}
```

Comment:
maybe we talked about this before, but is it null safe? Since the option is not required. I would rather expect something like helpers.empty

Reply:
Yes it is null save, I'm using it extensively in the whole template and it works as expected. If the BlancDesc is chosen, the ``level`` vanishes.

Reply:
ok fine :)